product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx

```diff
@@ -298,30 +298,31 @@ To restore a deleted cluster, use the `--from-deleted` flag in the command.
 You can restore a cluster in a single cluster to a primary/standby high-availability cluster and vice versa. You can restore a distributed high-availability cluster only to a cluster using the same architecture.
 !!!
 
-### Pausing a cluster
+### Pause a cluster
 
 To pause a cluster, use the `cluster pause` command. The `cluster pause` command supports `flag` or `interactive` mode. The syntax for the command is:
 
 ```
 biganimal cluster pause {--id | --provider --region --name}
 ```
 
-Where `id` is a valid cluster ID. The `id` is mandatory.
-      `provider` is a cloud provider of the cluster.
-      `region` is the region of the cluster.
-      `name` is the name of the cluster.
+Where:
+- `id` is a valid cluster ID. The `id` is mandatory.
```

Comment:
@nidhibhammar It doesn't seem that `id` is mandatory if you can instead use provider/region/name.